tests/performance_tests/performance_test_basic.py

```diff
@@ -18,6 +19,28 @@
 errorExit = Utils.errorExit
 cmdError = Utils.cmdError
 relaunchTimeout = 30
+emptyBlockGoal = 5
+
+def waitForEmptyBlocks(node1):
```

Comment:
we should probably make this generic `node` not specific since the method could be used on any node, and we just happen to call it on `node1`